docs/starters.yml

```
@@ -3728,3 +3728,13 @@
     - Email verification
     - Includes React Helmet to allow editing site meta tags
     - Includes plugins for offline support out of the box
+    
+- url: https://agility-gatsby-starter.netlify.com
+  repo: https://github.com/agility/agility-gatsby-starter
+  description: Get started with Gatsby and Agility CMS using a minimal blog.
+  tags:
+    - CMS:AgilityCMS
```

Please use cms:other for now

```suggestion
    - CMS:Other
```